tests/test_pcost.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from src.pcost import portfolio_cost
+
+
+def test_portfolio_cost():
+    """
+    Tests if portfolio_cost calculates the cost of a portfolio correctly.
+    """
+    assert portfolio_cost("data/portfolio.csv") == 44671.15
```

Comment:
When you are dealing with paths with Python you have 2 options:
- [os](https://docs.python.org/3/library/os.html)
- [pathlib](https://docs.python.org/3/library/pathlib.html)

This allows to prevent several common bugs. For example, in this case it won't work on Windows since the separator is different than Unix system.

Since Python 3.4+, pathlib is the standard way to deal with paths. In general, it is a good practice to define the directories that you will be using in `src/__init__.py`. Here you will use the `data` folder often so what you can do is:
```python
from pathlib import Path

CURRENT_DIRECTORY = Path(__file__).resolve().parent # path to src
ROOT_DIRECTORY = CURRENT_DIRECTORY.parent # path to directory which contains src and data

DATA_DIRECTORY = ROOT_DIRECTORY / "data"
```

And then you can import the `DATA_DIRECTORY` in the different modules. For example, for this test you will have:
```python
def test_portofio_cost():
    assert portfolio_cost(DATA_DIRECTORY / "portfolio.csv") == 44671.15
```

:warning: the type of (DATA_DIRECTORY / "portoflio.csv") will be a `Path` object and not a string. It will still work with `open` but you have to update the static typing from `str` to `Union[str, Path]`


------------

Here you can find 2 articles on pathlib:
- [Why you should be using pathlib](https://treyhunner.com/2018/12/why-you-should-be-using-pathlib/)
- [No really, pathlib is great](https://treyhunner.com/2019/01/no-really-pathlib-is-great/)